skimage/measure/tests/test_fit.py

```diff
@@ -195,16 +237,23 @@ def test_circle_model_estimate_from_small_scale_data():
         dtype=np.float64,
     )
     data = CircleModel().predict_xy(angles, params=params)
```

Comment:
Out of scope: but this seems like a weird API too. With the current signature `predict_xy` should be a classmethod. Alternatively, the `CircleModel`'s constructor should accept `params=`.